### ISSUE TITLE
Why there is not any usage example

### ISSUE THREAD

**Issue by Oxi84** [zihangdai/xlnet#176]
I appreciate you hard work on the language models, but I do not really understand why there is not any code example how to predict a masked word or hidden states at least using pretrained models.

Lets say that I have a sentence such as "This is my <mask> sentence".  
How do I predict distribution probability for word <mask>.

**Comment by hiwaveSupport** [comment]
yes this is sorely missing - just simple example using pre-trained models.
